i18n/management/commands/publish_i18n.py

```diff
@@ -19,7 +19,8 @@ class Command(BaseCommand):
     ]
 
     language_codes = [
-        language_code for language_code, _ in settings.LANGUAGES
+        (language_code, language_code in settings.LANGUAGE_GENERATE_PDF)
```

Comment:
is there a specific reason to add this here instead of inferring it explicitly below? I worry about a loss of readability

Reply:
Changed